spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java

```diff
@@ -81,6 +81,23 @@ private StringBuilder generateDashboardCell(StringBuilder b, String misoClass, L
     return b;
   }
 
+  private StringBuilder generateProjectDashboardCell(StringBuilder b, String misoClass, Long id, String name, String alias,
+      String shortName, String... aliasAlternative) {
+    b.append("<a class=\"dashboardresult\" href=\"/miso/" + misoClass + "/" + id
+        + "\"><div onMouseOver=\"this.className=&#39dashboardhighlight&#39\" onMouseOut=\"this.className=&#39dashboard&#39\" class=\"dashboard\">");
+    b.append("Name: <b>" + name + "</b><br/>");
+    b.append("Short Name: <b>" + shortName + "</b><br/>");
+    if (aliasAlternative == null || aliasAlternative.length == 0) {
```

Comment:
What sorts of alias alternatives might be expected?
The Short Name line should probably only be displayed if `shortName` is not null (it's an OICR-ism that we require but is optional for other MISO instances)

Reply:
I grabbed the alias alternative code from the old `generateDashboardCell` function, although I suppose it won't be necessary if this function is going to be specifically acting on pools.